Zilon.Core/Zilon.Core/Tactics/GameLoop.cs

```diff
@@ -1,87 +1,137 @@
 ﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
 using System.Linq;
+using System.Threading.Tasks;
 
-using Zilon.Core.Persons;
-using Zilon.Core.Players;
+using Zilon.Core.PersonModules;
 using Zilon.Core.Tactics.Behaviour;
 
 namespace Zilon.Core.Tactics
 {
-    public sealed class GameLoop : IGameLoop
+    public sealed partial class GameLoop : IGameLoop
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5544638)